FIX:fix error

mjseok opened the pull request:
fix error

Fixed errors in 'game_statement' variable.

함수 세분화할때 잘못 수정한 부분을 고침
game_statement의 결과가 올바르게 나오게 함.

mjseok committed:
FIX:fix error

fix error

Fixed errors in 'game_statement' variable.

seohyeonKK commented:
chooseSymbol()이 선언된 것과 정의된 형식이 다른거 같습니다!
그리구 current_player, game_start 변수가 current_plyaerm, game_statr라고 바뀌어져 있는것 같아요
확인 부탁드립니다~~

mjseok committed:
Fix:fix error

fix error
fix errors in 'game_statement' variables.
correct 'current_plyaer' and 'game_statr' typo.
